src/components/PagesList.js

```diff
@@ -0,0 +1,39 @@
+import glamorous from 'glamorous';
+import { Link } from 'phenomic';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+const List = glamorous.ul({
+  display: 'flex',
+  flexDirection: 'column',
+  listStyle: 'none',
+  padding: 0,
+});
+
+const PagesList = ({ pages }) => (
+  <div>
+    {pages.length ?
```

Comment:
Ternary operator is gross as heck but I'm not sure what parts of this you wrote.